repos/rust-analyzer/rustc-auto-publish.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-analyzer"
+name = "rustc-auto-publish"
+description = ""
+bots = []
+
+[access.teams]
+rust-analyzer = "write"
```

Comment:
With this I will lose access to the repo settings is that correct? Cause right now I kind of need access for managing the crates-io token there for publishing (whenever we need to publish-new something as the current token only has publish-update permissions)

Reply:
Good point. I changed it to "maintain", but I'm not sure if that will be enough. If not, we can discuss how to proceed with t-infra, because we'd like to avoid giving admin permissions, as these allow destroying repos or transferring them out of the org.